scripts/manage-hooks.sh

```
@@ -0,0 +1,435 @@
+    echo -e "${PURPLE}Recent hook-related commits:${NC}"
+    git log --oneline --grep="hook" --grep="lint" --grep="detekt" -10 2>/dev/null || echo "No hook-related commits found"
+}
+
+# Function to clean cache
+clean_cache() {
+    echo -e "${BLUE}🧹 Cleaning hook cache and temporary files...${NC}"
+
+    # Clean cache directory
+    if [ -d "$HOOK_CACHE_DIR" ]; then
+        cache_size=$(du -sh "$HOOK_CACHE_DIR" 2>/dev/null | cut -f1)
+        rm -rf "$HOOK_CACHE_DIR"/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)